File: CsAddition/testCalcVb/calcTest.cs
using CsAddition;
using Microsoft.VisualStudio.TestTools.UnitTesting;
using System;

namespace testCalcVb
{
    
    
    /// <summary>
    ///Classe de test pour calcTest, destinée à contenir tous
    ///les tests unitaires calcTest
    ///</summary>
    [TestClass()]
    public class calcTest
    {


        private TestContext testContextInstance;

        /// <summary>
        ///Obtient ou définit le contexte de test qui fournit
        ///des informations sur la série de tests active ainsi que ses fonctionnalités.
        ///</summary>
        public TestContext TestContext
        {
            get
            {
                return testContextInstance;
            }
            set
            {
                testContextInstance = value;
            }
        }

        #region Attributs de tests supplémentaires
        // 
        //Vous pouvez utiliser les attributs supplémentaires suivants lorsque vous écrivez vos tests :
        //
        //Utilisez ClassInitialize pour exécuter du code avant d'exécuter le premier test dans la classe
        //[ClassInitialize()]
        //public static void MyClassInitialize(TestContext testContext)
        //{
        //}
        //
        //Utilisez ClassCleanup pour exécuter du code après que tous les tests ont été exécutés dans une classe
        //[ClassCleanup()]
        //public static void MyClassCleanup()
        //{
        //}
        //
        //Utilisez TestInitialize pour exécuter du code avant d'exécuter chaque test
        //[TestInitialize()]
        //public void MyTestInitialize()
        //{
        //}
        //
        //Utilisez TestCleanup pour exécuter du code après que chaque test a été exécuté
        //[TestCleanup()]
        //public void MyTestCleanup()
        //{
        //}
        //
        #endregion


        /// <summary>
        ///Test pour Constructeur calc
        ///</summary>
        [TestMethod()]
        [DeploymentItem("CsAddition.exe")]
        public void calcConstructorTest()
        {
            double a = 0F; // TODO: initialisez à une valeur appropriée
            double b = 0F; // TODO: initialisez à une valeur appropriée
            calc_Accessor target = new calc_Accessor(a, b);
            Assert.Inconclusive("TODO: implémentez le code pour vérifier la cible");
        }

        /// <summary>
        ///Test pour Constructeur calc
        ///</summary>
        [TestMethod()]
        [DeploymentItem("CsAddition.exe")]
        public void calcConstructorTest1()
        {
            double a = 0F; // TODO: initialisez à une valeur appropriée
            double b = 0F; // TODO: initialisez à une valeur appropriée
            double c = 0F; // TODO: initialisez à une valeur appropriée
            calc_Accessor target = new calc_Accessor(a, b, c);
            Assert.Inconclusive("TODO: implémentez le code pour vérifier la cible");
        }

        /// <summary>
        ///Test pour Constructeur calc
        ///</summary>
        [TestMethod()]
        [DeploymentItem("CsAddition.exe")]
        public void calcConstructorTest2()
        {
            calc_Accessor target = new calc_Accessor();
            Assert.Inconclusive("TODO: implémentez le code pour vérifier la cible");
        }

        /// <summary>
        ///Test pour addition
        ///</summary>
        [TestMethod()]
        [DeploymentItem("CsAddition.exe")]
        public void additionTest()
        {
            calc_Accessor target = new calc_Accessor(); // TODO: initialisez à une valeur appropriée
            double a = 0F; // TODO: initialisez à une valeur appropriée
            double b = 0F; // TODO: initialisez à une valeur appropriée
            double c = 0F; // TODO: initialisez à une valeur appropriée
            double d = 0F; // TODO: initialisez à une valeur appropriée
            double expected = 0F; // TODO: initialisez à une valeur appropriée
            double actual;
            actual = target.addition(a, b, c, d);
            Assert.AreEqual(expected, actual);
            Assert.Inconclusive("Vérifiez l\'exactitude de cette méthode de test.");
        }

        /// <summary>
        ///Test pour addition
        ///</summary>
        [TestMethod()]
        [DeploymentItem("CsAddition.exe")]
        public void additionTest1()
        {
            calc_Accessor target = new calc_Accessor(); // TODO: initialisez à une valeur appropriée
            double expected = 0F; // TODO: initialisez à une valeur appropriée
            double actual;
            actual = target.addition();
            Assert.AreEqual(expected, actual);
            Assert.Inconclusive("Vérifiez l\'exactitude de cette méthode de test.");
        }

        /// <summary>
        ///Test pour addition
        ///</summary>
        [TestMethod()]
        [DeploymentItem("CsAddition.exe")]
        public void additionTest2()
        {
            calc_Accessor target = new calc_Accessor(); // TODO: initialisez à une valeur appropriée
            double a = 0F; // TODO: initialisez à une valeur appropriée
            double b = 0F; // TODO: initialisez à une valeur appropriée
            double c = 0F; // TODO: initialisez à une valeur appropriée
            double expected = 0F; // TODO: initialisez à une valeur appropriée
            double actual;
            actual = target.addition(a, b, c);
            Assert.AreEqual(expected, actual);
            Assert.Inconclusive("Vérifiez l\'exactitude de cette méthode de test.");
        }

        /// <summary>
        ///Test pour addition
        ///</summary>
        [TestMethod()]
        [DeploymentItem("CsAddition.exe")]
        public void additionTest3()
        {
            calc_Accessor target = new calc_Accessor(); // TODO: initialisez à une valeur appropriée
            string a = string.Empty; // TODO: initialisez à une valeur appropriée
            string b = string.Empty; // TODO: initialisez à une valeur appropriée
            string expected = string.Empty; // TODO: initialisez à une valeur appropriée
            string actual;
            actual = target.addition(a, b);
            Assert.AreEqual(expected, actual);
            Assert.Inconclusive("Vérifiez l\'exactitude de cette méthode de test.");
        }

        /// <summary>
        ///Test pour addition
        ///</summary>
        [TestMethod()]
        [DeploymentItem("CsAddition.exe")]
        public void additionTest4()
        {
            calc_Accessor target = new calc_Accessor(); // TODO: initialisez à une valeur appropriée
            double a = 0F; // TODO: initialisez à une valeur appropriée
            double b = 0F; // TODO: initialisez à une valeur appropriée
            double expected = 0F; // TODO: initialisez à une valeur appropriée
            double actual;
            actual = target.addition(a, b);
            Assert.AreEqual(expected, actual);
            Assert.Inconclusive("Vérifiez l\'exactitude de cette méthode de test.");
        }
    }
}

File: CsAddition/TestVb/calcTest.vb
Imports Microsoft.VisualStudio.TestTools.UnitTesting

Imports CsAddition



'''<summary>
'''Classe de test pour calcTest, destinée à contenir tous
'''les tests unitaires calcTest
'''</summary>
<TestClass()> _
Public Class calcTest


    Private testContextInstance As TestContext

    '''<summary>
    '''Obtient ou définit le contexte de test qui fournit
    '''des informations sur la série de tests active ainsi que ses fonctionnalités.
    '''</summary>
    Public Property TestContext() As TestContext
        Get
            Return testContextInstance
        End Get
        Set(value As TestContext)
            testContextInstance = Value
        End Set
    End Property

#Region "Attributs de tests supplémentaires"
    '
    'Vous pouvez utiliser les attributs supplémentaires suivants lorsque vous écrivez vos tests :
    '
    'Utilisez ClassInitialize pour exécuter du code avant d'exécuter le premier test dans la classe
    '<ClassInitialize()>  _
    'Public Shared Sub MyClassInitialize(ByVal testContext As TestContext)
    'End Sub
    '
    'Utilisez ClassCleanup pour exécuter du code après que tous les tests ont été exécutés dans une classe
    '<ClassCleanup()>  _
    'Public Shared Sub MyClassCleanup()
    'End Sub
    '
    'Utilisez TestInitialize pour exécuter du code avant d'exécuter chaque test
    '<TestInitialize()>  _
    'Public Sub MyTestInitialize()
    'End Sub
    '
    'Utilisez TestCleanup pour exécuter du code après que chaque test a été exécuté
    '<TestCleanup()>  _
    'Public Sub MyTestCleanup()
    'End Sub
    '
#End Region


    '''<summary>
    '''Test pour Constructeur calc
    '''</summary>
    <TestMethod(), _
     DeploymentItem("CsAddition.exe")> _
    Public Sub calcConstructorTest()
        Dim a As Double = 0.0! ' TODO: initialisez à une valeur appropriée
        Dim b As Double = 0.0! ' TODO: initialisez à une valeur appropriée
        Dim target As calc_Accessor = New calc_Accessor(a, b)
        Assert.Inconclusive("TODO: implémentez le code pour vérifier la cible")
    End Sub

    '''<summary>
    '''Test pour Constructeur calc
    '''</summary>
    <TestMethod(), _
     DeploymentItem("CsAddition.exe")> _
    Public Sub calcConstructorTest1()
        Dim a As Double = 0.0! ' TODO: initialisez à une valeur appropriée
        Dim b As Double = 0.0! ' TODO: initialisez à une valeur appropriée
        Dim c As Double = 0.0! ' TODO: initialisez à une valeur appropriée
        Dim target As calc_Accessor = New calc_Accessor(a, b, c)
        Assert.Inconclusive("TODO: implémentez le code pour vérifier la cible")
    End Sub

    '''<summary>
    '''Test pour Constructeur calc
    '''</summary>
    <TestMethod(), _
     DeploymentItem("CsAddition.exe")> _
    Public Sub calcConstructorTest2()
        Dim target As calc_Accessor = New calc_Accessor()
        Assert.Inconclusive("TODO: implémentez le code pour vérifier la cible")
    End Sub

    '''<summary>
    '''Test pour addition
    '''</summary>
    <TestMethod(), _
     DeploymentItem("CsAddition.exe")> _
    Public Sub additionTest()
        Dim target As calc_Accessor = New calc_Accessor() ' TODO: initialisez à une valeur appropriée
        Dim a As Double = 0.0! ' TODO: initialisez à une valeur appropriée
        Dim b As Double = 0.0! ' TODO: initialisez à une valeur appropriée
        Dim c As Double = 0.0! ' TODO: initialisez à une valeur appropriée
        Dim d As Double = 0.0! ' TODO: initialisez à une valeur appropriée
        Dim expected As Double = 0.0! ' TODO: initialisez à une valeur appropriée
        Dim actual As Double
        actual = target.addition(a, b, c, d)
        Assert.AreEqual(expected, actual)
        Assert.Inconclusive("Vérifiez l'exactitude de cette méthode de test.")
    End Sub

    '''<summary>
    '''Test pour addition
    '''</summary>
    <TestMethod(), _
     DeploymentItem("CsAddition.exe")> _
    Public Sub additionTest1()
        Dim target As calc_Accessor = New calc_Accessor() ' TODO: initialisez à une valeur appropriée
        Dim expected As Double = 0.0! ' TODO: initialisez à une valeur appropriée
        Dim actual As Double
        actual = target.addition
        Assert.AreEqual(expected, actual)
        Assert.Inconclusive("Vérifiez l'exactitude de cette méthode de test.")
    End Sub

    '''<summary>
    '''Test pour addition
    '''</summary>
    <TestMethod(), _
     DeploymentItem("CsAddition.exe")> _
    Public Sub additionTest2()
        Dim target As calc_Accessor = New calc_Accessor() ' TODO: initialisez à une valeur appropriée
        Dim a As Double = 0.0! ' TODO: initialisez à une valeur appropriée
        Dim b As Double = 0.0! ' TODO: initialisez à une valeur appropriée
        Dim c As Double = 0.0! ' TODO: initialisez à une valeur appropriée
        Dim expected As Double = 0.0! ' TODO: initialisez à une valeur appropriée
        Dim actual As Double
        actual = target.addition(a, b, c)
        Assert.AreEqual(expected, actual)
        Assert.Inconclusive("Vérifiez l'exactitude de cette méthode de test.")
    End Sub

    '''<summary>
    '''Test pour addition
    '''</summary>
    <TestMethod(), _
     DeploymentItem("CsAddition.exe")> _
    Public Sub additionTest3()
        Dim target As calc_Accessor = New calc_Accessor() ' TODO: initialisez à une valeur appropriée
        Dim a As String = String.Empty ' TODO: initialisez à une valeur appropriée
        Dim b As String = String.Empty ' TODO: initialisez à une valeur appropriée
        Dim expected As String = String.Empty ' TODO: initialisez à une valeur appropriée
        Dim actual As String
        actual = target.addition(a, b)
        Assert.AreEqual(expected, actual)
        Assert.Inconclusive("Vérifiez l'exactitude de cette méthode de test.")
    End Sub

    '''<summary>
    '''Test pour addition
    '''</summary>
    <TestMethod(), _
     DeploymentItem("CsAddition.exe")> _
    Public Sub additionTest4()
        Dim target As calc_Accessor = New calc_Accessor() ' TODO: initialisez à une valeur appropriée
        Dim a As Double = 5.0! ' TODO: initialisez à une valeur appropriée
        Dim b As Double = 3.0! ' TODO: initialisez à une valeur appropriée
        Dim expected As Double = 9.0! ' TODO: initialisez à une valeur appropriée
        Dim actual As Double
        actual = target.addition(a, b)
        Assert.AreEqual(expected, actual)
        'Assert.Inconclusive("Vérifiez l'exactitude de cette méthode de test.")  // ne peut pas tirer de conclusion
    End Sub
End Class
